### ISSUE TITLE
[Suggestion]: Anti Drowning

### ISSUE THREAD

**Issue by Mezimaru** [Mrbysco/ChocoCraft4#66]
### General Info

- [ ] I would like to propose new content
- [X] I would like to propose changes
- [ ] I would like to propose another thing

### Suggestion Description

I'd still love some way to prevent Chocobo submerged in water from drowning if dismounted when they're under the stay command.
I personally am currently using 

		if (this.isUnderWater()) {
			setNoAi(false);
			this.goalSelector.addGoal(0, this.follow);
			this.setMovementType(MovementType.FOLLOW_OWNER);
		}

But that's probably quite rudimentary, I'm sure you have another method you'd rather utilize

Love all the work you're doing <3

**Comment by Mrbysco** [comment]
I rather not attach an AI goal every tick that the chocobo is underwater and never remove the goal

**Comment by Mezimaru** [comment]
Hahaha yeah, which is why I said it was a quite rudimentary way of doing it ;P

Also, testing, I notice flying chocobo cannot submerge/swim the same way the blue+ chocobo can. Not sure if its a problem or anything, just something I noticed.

**Comment by Mrbysco** [comment]
I'm currently checking if the chocobo can stand on water because the chocobo's that can stand on water should also be able to swim more easily than the chocobo's that cant

**Comment by Mezimaru** [comment]
Right, but for example, you can't submerge in water and swim with a gold
chocobo

**Comment by Mezimaru** [comment]
Also looks like yellow/green Chocobo are still stuck continuously jumping on the water surface if your holding jump and continue to hold jump for a second or two when jumping into the water.

Just tinkering around, commenting out the jumping below seemed to fix it for me, also was unable to determine any detriment from commenting it out in playtesting

else if (onGround()) {
jumpFromGround();
livingentity.setJumping(false);
//this.setJumping(true);
}

(Sorry, you're probably sick of me by now)
